Source/ProceduralFairings/FairingBase.cs

```diff
@@ -117,6 +121,9 @@ public enum BaseMode { Payload, Adapter, Plate, Other }
         readonly List<ConfigurableJoint> joints = new List<ConfigurableJoint>();
         public DragCubeUpdater dragCubeUpdater;
         public ModuleDecouple Decoupler;
+        private readonly List<ProceduralFairingSide> sides = new List<ProceduralFairingSide>();
+        public Vector3 editorOpenOffset = new Vector3(3, 0, 0);
+        private float minOffset = 2f;
```

Comment:
Trick:
`public Vector3 EditorOpenOffset => new Vector3(OffsetValueX, 0, 0);`

Then you won't have to mess with trying to set the x value manually.  Whenever you ask for the EditorOpenOffset Vector3, you will get the appropriate one.

Declare `minOffset` as `const` ie `private const float minOffset = 2`